ports/nrf/common-hal/audiobusio/I2SOut.c

```diff
@@ -139,14 +139,14 @@ static void i2s_buffer_fill(audiobusio_i2sout_obj_t* self) {
             uint16_t *bp = (uint16_t*)buffer;
             uint16_t *be = (uint16_t*)(buffer + bytecount);
             uint16_t *sp = (uint16_t*)self->sample_data;
-            for (; bp != be; bp++) {
+            for (; bp != be;) {
```

Comment:
I would use `<` in loops like this on general safety principles.

Reply:
Good call.  Using only != is a bad habit I picked up while working with C++, where it is necessary for certain container types.  I'll try to remember and break it.